### ISSUE TITLE
Move setting the  Username and Password to an in UI config flow

### ISSUE THREAD

**Issue by jcronq** [apetrycki/daikinskyport#5]
Currently setting username and password from configuration.yaml. This would be better moved to the UI configuration step so Home Assisstant can manage the secrets appropriately.

**Comment by apetrycki** [comment]
I'm working on this one and will hopefully have something to test out this weekend.  I'll need some people to make sure I didn't miss or break anything.  This should also roll up some other issues as well.

**Comment by apetrycki** [comment]
I've made good progress on this.  There are a few bugs that I could use some help squashing.  For some reason, all the entities only show up after a restart of HA.  I also for the life of me cannot figure out why the labels are not showing up on the config flow dialog.  The strings.json looks correct as far as I can tell.  I've looked over the documentation several times and still can't figure it out.  Check out the ui_config branch.  I also haven't tried creating a migration for configuration.yaml yet, but I don't think that's really a big deal.
